gallery/src/main.scss

```diff
@@ -4,80 +4,80 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-@import '~@sonatype/react-shared-components/scss-shared/nx-variables';
-@import '~@sonatype/react-shared-components/scss-shared/nx-text-helpers';
-@import '~@sonatype/react-shared-components/scss-shared/nx-container-helpers';
-@import '~@sonatype/react-shared-components/scss-shared/nx-flexbox-helpers';
+ @import '~@sonatype/react-shared-components/scss-shared/nx-variables';
```

Comment:
Bad formatting changes throughout this file

Reply:
> Bad formatting changes throughout this file

Fixed in 3f37a3b4c8e43826e6a666b724a7374662947440